README.md

```diff
@@ -69,29 +69,31 @@ Only the `source` import source phase is specified by this proposal.
 
 ### Defining Module Source
 
-Module source is defined through the [ECMA-262 ES modules HostLoadImportedModule refactoring][],
-which moves the construction of module records to the host, and on which we can define a
-`[[ModuleSourceObject]]` custom object to be returned by module source phase that is compatible
-with module expressions.
+The object provided by the module source phase must have a prototype set to the
+`ModuleSourcePrototype` object, defined by this specification to be a minimal
+shared base prototype for a compiled modular resource.
+
+In addition we define the `sourceType` getter returning the type string
+corresponding to the name of the specific module source subclass.
```

Comment:
```suggestion
The object provided by the module source phase must have a the
`AbstractModuleSourcePrototype` object in its prototype chain, defined by this
specification to be a minimal shared base prototype for a compiled modular resource.
```

Reply:
Also include something along the lines of:

For now, the `%AbtractModuleSourcePrototype%` contains only a `@toStringTag` getter.